external/mbedtls/alt/entropy_poll_alt.c

```diff
@@ -38,18 +34,23 @@ static int mbedtls_generate_random_alt(unsigned char *data, unsigned int len)
 {
 	int ret;
 	sl_ctx shnd;
-	hal_data random = {data, len, NULL, 0};
 
+	hal_data random = {data, len, NULL, 0};
+	hal_result_e hres = HAL_FAIL;
 	ret = sl_init(&shnd);
 	if (ret != SECLINK_OK) {
 		return MBEDTLS_ERR_ENTROPY_SOURCE_FAILED;
 	}
 
-	ret = sl_generate_random(shnd, len, &random);
-	sl_deinit(shnd);
-	if (ret != SECLINK_OK) {
+	ret = sl_generate_random(shnd, len, &random/*, &hres*/);
+
+	if ((ret != SECLINK_OK) /*|| (hres != HAL_SUCCESS)*/) {
+		sl_deinit(shnd);
```

Comment:
Let's erase `hres`

Reply:
Not only for `entropy_poll_alt.c`, `hres` still left in some `_alt` codes. Please erase them.

Reply:
removed hres from all the _alt files.